[SOAR-4789] ICON Template Migration (#138)
rapid7/icon-integrations-validators: README.md

## Changelog

> * 2.41.0 - Convert dataclass to pydantic for Workflow to use pydantic | Update icon-validate to validate with template validators only | Move validate_id and validate_workflow_version to icon validators

Reviewer: ```suggestion
* 2.41.0 - Convert dataclass to pydantic for Workflow to use pydantic | Update icon-validate to add template validation capabilities
```
